code/src/reducers/gameInfo.js

```diff
@@ -0,0 +1,40 @@
+import { gameDetails } from './gameDetails';
+
+export const getDescription = (userName) => {
+  return (dispatch, getState) => {
+    dispatch(gameDetails.actions.setLoading(true));
+    fetch('https://wk16-backend.herokuapp.com/start', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({ username: userName })
```

Comment:
If you would name your argument in line 3 as `username` (without camelCase), then we could simplify this line of code :

```
body: JSON.stringify({ username })
```